### ISSUE TITLE
Show available demand when making a slot

### ISSUE THREAD

**Issue by Ivshti** [heyAuraDev/adex-platform#406]
**Is your feature request related to a problem? Please describe.**
When making an ad slot, the user does not know how much demand there is for this slot, except the estimation shown when picking a size (#106).

**Describe the solution you'd like**
If there's no demand for this ad slot, warn the user and explain why ("no campaigns for this size", "no campaigns satisfying this CPM requirement", "no campaigns targeted for your audience").

basically, this explains "why ads won't show" (together with #394)

**Additional context**
May require implementing https://github.com/AdExNetwork/adex-market/issues/91 first

**Comment by Ivshti** [comment]
Seems to be done
![Screenshot_2020-06-03 AdEx Platform](https://user-images.githubusercontent.com/1247629/83615557-63b60100-a58f-11ea-8b73-89116d6803ba.png)

**Comment by Ivshti** [comment]
This issue is a bit different in that it should also consider categories, min CPMs and etc.
